src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -214,907 +214,908 @@ void GetPurchasedAir(EnergyPlusData &state)
-            PurchAir(PurchAirNum).LatHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).TotHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).SenCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).LatCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).TotCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).ZoneSenHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).ZoneLatHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).ZoneTotHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).ZoneSenCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).ZoneLatCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).ZoneTotCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).OASenHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).OALatHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).OATotHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).OASenCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).OALatCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).OATotCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).HtRecSenHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).HtRecLatHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).HtRecTotHeatEnergy = 0.0;
-            PurchAir(PurchAirNum).HtRecSenCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).HtRecLatCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).HtRecTotCoolEnergy = 0.0;
-            PurchAir(PurchAirNum).SenHeatRate = 0.0;
-            PurchAir(PurchAirNum).LatHeatRate = 0.0;
-            PurchAir(PurchAirNum).TotHeatRate = 0.0;
-            PurchAir(PurchAirNum).SenCoolRate = 0.0;
-            PurchAir(PurchAirNum).LatCoolRate = 0.0;
-            PurchAir(PurchAirNum).TotCoolRate = 0.0;
-            PurchAir(PurchAirNum).ZoneSenHeatRate = 0.0;
-            PurchAir(PurchAirNum).ZoneLatHeatRate = 0.0;
-            PurchAir(PurchAirNum).ZoneTotHeatRate = 0.0;
-            PurchAir(PurchAirNum).ZoneSenCoolRate = 0.0;
-            PurchAir(PurchAirNum).ZoneLatCoolRate = 0.0;
-            PurchAir(PurchAirNum).ZoneTotCoolRate = 0.0;
-            PurchAir(PurchAirNum).OASenHeatRate = 0.0;
-            PurchAir(PurchAirNum).OALatHeatRate = 0.0;
-            PurchAir(PurchAirNum).OATotHeatRate = 0.0;
-            PurchAir(PurchAirNum).OASenCoolRate = 0.0;
-            PurchAir(PurchAirNum).OALatCoolRate = 0.0;
-            PurchAir(PurchAirNum).OATotCoolRate = 0.0;
-            PurchAir(PurchAirNum).HtRecSenHeatRate = 0.0;
-            PurchAir(PurchAirNum).HtRecLatHeatRate = 0.0;
-            PurchAir(PurchAirNum).HtRecTotHeatRate = 0.0;
-            PurchAir(PurchAirNum).HtRecSenCoolRate = 0.0;
-            PurchAir(PurchAirNum).HtRecLatCoolRate = 0.0;
-            PurchAir(PurchAirNum).HtRecTotCoolRate = 0.0;
-
-            PurchAir(PurchAirNum).OutdoorAirMassFlowRate = 0.0;
-            PurchAir(PurchAirNum).OutdoorAirVolFlowRateStdRho = 0.0;
-            PurchAir(PurchAirNum).SupplyAirMassFlowRate = 0.0;
-            PurchAir(PurchAirNum).SupplyAirVolFlowRateStdRho = 0.0;
```

Comment:
Deleted redundant code. These member variables are already initialized in the `PurchasedAirManager.hh` file.